docs/codeql/codeql-cli/using-custom-queries-with-the-codeql-cli.rst

```diff
@@ -33,8 +33,8 @@ following two properties to ensure that the results are interpreted correctly:
 
 - Query identifier (``@id``): a sequence of words composed of lowercase letters or
   digits, delimited by ``/`` or ``-``, identifying and classifying the query. 
-- Query type (``@kind``): identifies the query is an alert (``@kind problem``)
-  or a path (``@kind path-problem``).
+- Query type (``@kind``): identifies the query is an alert (``@kind problem``),
+  a path (``@kind path-problem``), or a diagnostic metric (``@kind diagnostic``).
```

Comment:
Might as well take this opportunity to fix the not very helpful description of path-problem here:
```suggestion
- Query type (``@kind``): identifies the query is a simple alert (``@kind problem``),
  an alert documented by a sequence of code locations (``@kind path-problem``), or a diagnostic metric (``@kind diagnostic``).
```

Reply:
There's one more kind: a query with `@kind metric` with `@tags summary` is considered a "summary metric" query.
For `@kind diagnostic` I would use the term "extractor diagnostic" or simply "diagnostic", rather than "diagnostic metric".